build.xml

```diff
@@ -312,33 +425,23 @@
       deployRoot="${source} sprint ${version}" />
   </target>
 
-
-  <!-- SYNTAX: ant local2prod-specified-tests -Dversion=3.6 -Dsource=qa -DspecifiedTests=PresalesRequestHandler_Test   optional: -DcheckOnly=true -DrunAllTests=true -->
+  <!-- SYNTAX: ant local2prod-specified-tests -Dversion=3.6 -Dsource=qa -DspecifiedTests=TestClass1,TestClass2   optional: -DcheckOnly=true -DrunAllTests=true -->
   <target name="local2prod-specified-tests">
-    <!-- backup prod first -->
-    <antcall target="backup-prod">
-      <param name="version" value="${version}" />
-      <param name="type" value="package" />
-    </antcall>
-    <antcall target="backup-prod">
-      <param name="version" value="${version}" />
-      <param name="type" value="destructiveChanges" />
-    </antcall>
-    <antcall target="copy-destructiveChanges">
-      <param name="version" value="${version}" />
-      <param name="source" value="${source}" />
-    </antcall>
-    <sf:deploy
-      username="${cmc.prod.username}"
-      password="${cmc.prod.password}"
-      serverurl="${prod.serverurl}"
-      pollWaitMillis="20000"
-      maxPoll="5000"
-      checkOnly="${checkOnly}"
-      testLevel="RunSpecifiedTests"
-      deployRoot="${source} sprint ${version}">
-      <runTest>${specifiedTest}</runTest>  <!-- PresalesRequestHandler_Test -->
-    </sf:deploy>
+      <!-- backup prod first -->
+      <antcall target="backup-prod">
+        <param name="version" value="${version}" />
+        <param name="type" value="package" />
+      </antcall>
+      <antcall target="backup-prod">
+        <param name="version" value="${version}" />
+        <param name="type" value="destructiveChanges" />
+      </antcall>
+      <antcall target="copy-destructiveChanges">
+        <param name="version" value="${version}" />
+        <param name="source" value="${source}" />
+      </antcall>
+  	<!-- deploy script -->
+  	<scepcificTest cmcuser="${cmc.prod.username}" cmcpass="${cmc.prod.password}" serverurl="${prod.serverurl}"/>

```

Comment:
Typo "scepcificTest".  Update names of username/password attributes per earlier comments.